main.go

```diff
@@ -1,4 +1,4 @@
-package main
+package presentswitch
```

Comment:
This change has a big implication. This means we are converting this repo as a Go library instead of a Go program. Is that what we want to do? I'm okay with that but be aware there are a few differences and maybe some diferences.

Reply:
I'm a go n00b -.-. Can we have it main and still have different files? All assuming package main? I'll do it then :p

Reply:
Yes, we can still have many files under this package all working as the main package. 

For this case, I think a program is more adequate than a library :)